pages/Table.tsx

```diff
@@ -1,5 +1,4 @@
 import React from "react";
-import "./Table.css";
```

Comment:
This is not page should not be here. https://nextjs.org/docs/pages/building-your-application/routing/pages-and-layouts

Reply:
done